src/main/java/com/gmail/nossr50/util/EventUtils.java

```diff
@@ -150,6 +151,13 @@ public static McMMOPlayerAbilityActivateEvent callPlayerAbilityActivateEvent(Pla
         return event;
     }
 
+    public static McMMOPlayerProfileLoadEvent callPlayerProfileLoadEvent(Player player, PlayerProfile profile){
+        McMMOPlayerProfileLoadEvent e = new McMMOPlayerProfileLoadEvent(player, profile);
```

Comment:
not a fan of shorthand one letter variable names, would prefer a descriptive var name instead of a single character